tests/end-to-end/director/integration_system_api_test.go

```diff
@@ -0,0 +1,157 @@
+
+	require.NotEmpty(t, intSysOutput)
+	assert.Equal(t, name, intSysOutput.Name)
+	saveQueryInExamples(t, getIntegrationSystemRequest.Query(), "query integration system")
+}
+
+func TestUpdateIntegrationSystem(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+	name := "int-system"
+	newName := "new-int-system"
+
+	t.Log("Create integration system")
+	intSys := createIntegrationSystem(t, ctx, name)
+	require.NotEmpty(t, intSys)
+
+	intSysInput := graphql.IntegrationSystemInput{Name: newName}
+	intSysGQL, err := tc.graphqlizer.IntegrationSystemInputToGQL(intSysInput)
+	updateIntegrationSystemRequest := fixUpdateIntegrationSystemRequest(intSys.ID, intSysGQL)
+	updateOutput := graphql.IntegrationSystemExt{}
+
+	// WHEN
+	t.Log("Update integration system")
+	err = tc.RunOperation(ctx, updateIntegrationSystemRequest, &updateOutput)
+	require.NoError(t, err)
+	require.NotEmpty(t, updateOutput.ID)
+	defer deleteIntegrationSystem(t, ctx, updateOutput.ID)
+
+	//THEN
+	t.Log("Check if Integration System was updated")
+	assert.Equal(t, newName, updateOutput.Name)
+	saveQueryInExamples(t, updateIntegrationSystemRequest.Query(), "update integration system")
+}
+
+func TestDeleteIntegrationSystem(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+	name := "int-system"
+
+	t.Log("Create integration system")
+	intSys := createIntegrationSystem(t, ctx, name)
+	require.NotEmpty(t, intSys)
+
+	deleteIntegrationSystemRequest := fixDeleteIntegrationSystem(intSys.ID)
+	deleteOutput := graphql.IntegrationSystemExt{}
+
+	// WHEN
+	t.Log("Delete integration system")
+	err := tc.RunOperation(ctx, deleteIntegrationSystemRequest, &deleteOutput)
+	require.NoError(t, err)
+
+	//THEN
+	t.Log("Check if Integration System was deleted")
+
+	out := getIntegrationSystem(t, ctx, intSys.ID)
+
+	require.Empty(t, out)
+	saveQueryInExamples(t, deleteIntegrationSystemRequest.Query(), "delete integration system")
+}
+
+func TestQueryIntegrationSystem(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+	name := "int-system"
+
+	t.Log("Create integration system")
+	intSys := createIntegrationSystem(t, ctx, name)
+	require.NotEmpty(t, intSys)
+
+	getIntegrationSystemRequest := fixIntegrationSystemRequest(intSys.ID)
+	output := graphql.IntegrationSystemExt{}
+
+	// WHEN
+	t.Log("Get integration system")
+	err := tc.RunOperation(ctx, getIntegrationSystemRequest, &output)
+	require.NoError(t, err)
+	require.NotEmpty(t, output.ID)
+	defer deleteIntegrationSystem(t, ctx, output.ID)
+
+	//THEN
+	t.Log("Check if Integration System was received")
+	assert.Equal(t, name, output.Name)
+	saveQueryInExamples(t, getIntegrationSystemRequest.Query(), "get integration system")
+}
+
+func TestQueryIntegrationSystems(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+	name1 := "int-system-1"
+	name2 := "int-system-2"
+
+	t.Log("Create integration systems")
+	intSys1 := createIntegrationSystem(t, ctx, name1)
+	require.NotEmpty(t, intSys1)
```

Comment:
If it's even possible that `createIntegrationSystem` returns empty struct (?), I think this `require` should be moved inside that function because I can't think of any situation when creating empty struct would be expected behavior.